### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ik-web**
[DEMO LINK](https://ik-web.github.io/react_tabs-with-router/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,48 +1,44 @@
...
-    <div className="section">
-      <h1 className="title">Home page</h1>
-      <h1 className="title">Tabs page</h1>
-      <h1 className="title">Page not found</h1>
+  <div className="App container">
+    <nav className="navbar">
+      <NavLink
+        className={({ isActive }) => (
+          `navbar-item is-tab ${isActive ? ' is-active' : ''}`
+        )}
```
Comment (jstmpelowycz) [inline src/App.tsx:18]: you'd rather apply `classnames`
```suggestion
        className={({ isActive }) => classNames(
          'navbar-item is-tab',
          { isActive },
        )}
```

File: src/App.tsx [hunk 2]
```diff
@@ -1,48 +1,44 @@
...
-          </li>
-          <li data-cy="tab">
-            <a href="#/">Tab 3</a>
-          </li>
-        </ul>
-      </div>
+      <NavLink
+        className={({ isActive }) => (
+          `navbar-item is-tab ${isActive ? ' is-active' : ''}`
+        )}
```
Comment (jstmpelowycz) [inline src/App.tsx:27]: here as well

File: src/App.tsx [hunk 3]
```diff
@@ -1,48 +1,44 @@
...
-        Please select a tab
-      </div>
-    </div>
-  </>
+    <main>
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="tabs/:tabId" element={<TabsPage tabs={tabs} />} />
+        <Route path="/tabs" element={<TabsPage tabs={tabs} />} />
+      </Routes>
```
Comment (jstmpelowycz) [inline src/App.tsx:39]: use nested routes
for more info, learn this article – https://reactrouter.com/docs/en/v6/getting-started/concepts#index-routes

Comment (jstmpelowycz) [reply src/App.tsx:39]: also – don't forget about it
Screen Shot 2022-08-05 at 09 23 20

**Review by jstmpelowycz — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (ik-web) [reply src/App.tsx]: Fixed

File: src/App.tsx (hunk 2 above)
Comment (ik-web) [reply src/App.tsx]: Fixed

File: src/App.tsx (hunk 3 above)
Comment (ik-web) [reply src/App.tsx]: Fixed

**Review by mykola-koval — APPROVED** [review]

**Pull request closed without merging**